cupsfilters/image.c

```diff
@@ -871,3 +874,121 @@ get_tile(cf_image_t *img,		/* I - Image */
 
   return (ic->pixels + bpp * (y * CF_TILE_SIZE + x));
 }
+
+/*
+  helper function required by EXIF read function
+  */
+
+void trim_spaces(char *buf)
```

Comment:
Use `static void trim_spaces(...)` here, as this function is local to only this file.

Reply:
I should not declare local functions in image-private header file. right?

Reply:
Functions which are used only in a single file, in the file where they are defined, do not need to be mentioned in any `*.h` file. The `...-private.h` files are for functions which are not part of the API of the library but which are used in more than one source file of the library. So `trim_spaces()` does not need to be put into any `*.h` file.

Reply:
Yeah, got it

Reply:
I have made desired changes in commits below. Please have a look.